src/index.js

```diff
@@ -340,11 +328,17 @@ export default class HyperScreen extends React.Component {
 
   /**
    * Reload if an error occured.
+   * @param opt_href: Optional string href to use when reloading the screen. If not provided,
+   * the screen's current URL will be used.
    */
-  reload = () => {
+  reload = (opt_href) => {
```

Comment:
I think you need to use camel case and flow type annotations for linting to succeed